### PR TITLE
Param appium

### PR DESCRIPTION
**Pull request by Infobeans-Anujita**
The conftest.py file contains all the changes which parameterizes all the arguments from command line.

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -8,27 +8,44 @@ def pytest_addoption(parser):
     parser.addoption("--device", action="store", default="Android Emulator",
```
Comment (ersuniljoshi) [inline conftest.py]: ReadMe not updated, please send it.

File: conftest.py [hunk 2]
```diff
@@ -8,27 +8,44 @@ def pytest_addoption(parser):
...
                      help="my option: Android Emulator or Android device")
```
Comment (ersuniljoshi) [inline conftest.py]: Please run pep8 and flake8 on your code.

File: conftest.py (hunk 1 above)
Comment (Infobeans-Anujita) [inline conftest.py]: Done

File: conftest.py (hunk 2 above)
Comment (Infobeans-Anujita) [inline conftest.py]: Done

**Pull request merged**